src/maturity/index.md

```diff
@@ -0,0 +1,15 @@
+# What is a maturity rating?
+
+Before you unhide your game and show it to the world, you have to give it a maturity rating. Assign your game a rating, then go through the list and select any content tags that apply to your game.
```

Comment:
replace "Before you unhide your game and show it to the world" with "Maturity Ratings are required"